packages/scandipwa/src/component/MyAccountCreateAccount/MyAccountCreateAccount.component.js

```diff
@@ -125,7 +125,7 @@ export class MyAccountCreateAccount extends PureComponent {
 
         return (
             <fieldset block="MyAccountOverlay" elem="Legend">
-                <legend>{ __('Sign-Up Information') }</legend>
+                <legend block="Legend">{ __('Sign-Up Information') }</legend>
```

Comment:
That is meaningless. Block should be MyAccountOverlay, it can be element Legend or any other semantically correct one